### DISCUSSION TITLE
please help me activate the benefit!

### DISCUSSION THREAD

**Discussion by namsh1125** [community/community#131020] [GitHub Education]
### Hi everyone,

![image](https://github.com/community/community/assets/45845547/92a52be8-e6aa-4c74-958d-ec92e5e85e6f)

It was approved, but benefit is not activated even though some time has passed.

Please activate it 🙇‍♂️

**Comment by Sandeep2042001** [comment]
Hello @namsh1125 I am Sandeep,
Check yourself with the following Questions?
 1) Please check that when you did it your location was on.
 2) Did you Verify your email id?
 
 Feel free to reach out --- always.
